tests/jobs/test_scheduler_job.py

```diff
@@ -2909,13 +2922,7 @@ def complete_one_dagrun():
         # Pre-condition
         assert DagRun.active_runs_of_dags(session=session) == {'test_dag': 3}
 
-        assert model.next_dagrun == timezone.convert_to_utc(
-            timezone.DateTime(
-                2016,
-                1,
-                3,
-            )
-        )
+        assert model.next_dagrun == timezone.convert_to_utc(timezone.DateTime(2016, 1, 3, tzinfo=UTC))
```

Comment:
Isn't the double convert? `timezone.DateTime(2016, 1, 3, tzinfo=UTC)` should already give UTC the `convert_to_utc` is redundant isn't it?

Reply:
Actually yes :)

Reply:
Fixed